src/routes/Accounts/AccountDetails.tsx

```diff
@@ -34,10 +37,10 @@ const AccountDetails: FC = () => {
           label={'Back to all accounts'}
         />
         <Flex alignItems="end" mb="0.5rem">
-          <chakra.h2 mr="1rem">Primary Account</chakra.h2>
+          <chakra.h2 mr="1rem">{account?.name}</chakra.h2>
```

Comment:
What about loading state?

Reply:
Seems that we need to attach data loading fist so Skylar could have a look and do some suggestions.